File: internal/content/testdata/bench/go/nc-two-sum.go
func two_sum(nums []int, target int) []int {
	seen := map[int]int{}
	for i, n := range nums {
		need := target - n
		if j, ok := seen[need]; ok {
			return []int{j, i}
		}
		seen[n] = i
	}
	return []int{}
}

File: internal/content/testdata/bench/typescript/nc-two-sum.ts
function two_sum(nums: number[], target: number): number[] {
    const seen: {[key: number]: number} = {};
    for (let i = 0; i < nums.length; i++) {
        const need = target - nums[i];
        if (need in seen) {
            return [seen[need], i];
        }
        seen[nums[i]] = i;
    }
    return [];
}
